var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -33,51 +33,73 @@ class Subversion(Package):
     version('1.8.13',    '8065b3698d799507fb72dd7926ed32b6')
     version('1.9.3',     'a92bcfaec4e5038f82c74a7b5bbd2f46')
 
+    variant('python', default=False, description='Provide python bindings')
+    variant('perl', default=False, description='Provide Perl bindings')
+    # variant('ruby', default=False, description='Provide ruby bindings')
+
     depends_on('apr')
     depends_on('apr-util')
     depends_on('zlib')
     depends_on('sqlite')
     depends_on('serf')
+    depends_on('expat')
+    depends_on('swig@:2.99')
+    depends_on('autoconf', type='build')
```

Comment:
Why were these dependencies added unconditionally?  I presume `swig` is needed for the Python bindings.  These should have `when='+python'` or `when='+perl'` on them.  At the end of the day, this PR should add NO additional dependencies for the user who is not asking for Python or Perl bindings.